issue-repro-compose/src/main/AndroidManifest.xml

```diff
@@ -1,6 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="com.airbnb.lottie.issues.compose">
```

Comment:
setting `package` in `AndroidManifest` is now deprecated and produces a build warning